lambdas/indexer/app.py

```diff
@@ -463,3 +464,23 @@ def nudge(event: chalice.app.CloudWatchEvent):
         log.info('Nudging queue with %i token(s) for a total value of %i', len(tokens), num_documents)
         for batch in chunked(tokens, 10):
             _send_tokens(batch)
+
+
+@app.schedule('rate(5 minutes)')
+def retrieve_fail_messages(event_: chalice.app.CloudWatchEvent):
```

Comment:
I apologize for this again. Correction, is actually `_event` and you should see the linter warning disappear.